src/components/DashboardFilters/TimePeriodFilter/TimePeriodFilter-container.tsx

```diff
@@ -1,29 +1,70 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import Grid from '@material-ui/core/Grid/Grid';
-import { withStyles } from '@material-ui/core/styles';
 import Slider from '@material-ui/core/Slider';
+import Typography from '@material-ui/core/Typography';
 
 import { setDateToday, setDateYesterday } from '../helpers';
-import { TimePeriodStyles, useTimeFilterStyles } from './TimePeriodFilter-styles';
+import { useTimeFilterStyles } from './TimePeriodFilter-styles';
 
-const TimePeriodFilter = () => {
 
-  const TimePeriodSlider = withStyles(TimePeriodStyles)(Slider);
+const TimePeriodFilter = (props: any) => {
+
   const timeFilterClasses = useTimeFilterStyles();
 
   const today = setDateToday();
   const yesterday = setDateYesterday();
-
-  const setTimeFilter = () => {
-    const today = new Date();
-    return today.getHours() + ":" + today.getMinutes();
+  
+  const [timeStep, setTimeStep] : [number, Function] = useState(60000);
+  
+  const setTimeNow = () =>  Number(new Date().getTime() - new Date().getTimezoneOffset() * 60 * 1000);
+  const convertTimestampToTime = (timestamp: number) => new Date(timestamp).toISOString().substr(0, 19).slice(11, -3);
+  
+  const [timeValueTo, setTimeValueTo] : [number, Function] = useState(setTimeNow());
+  const [timeValueFrom, setTimeValueFrom] : [number, Function] = useState(setTimeNow() - timeStep);
+  
+  const setMinValue = () =>  Number(setTimeNow() - 24*60*60*1000);
```

Comment:
Może te funkcje do jakichś utilsów to będzie w contenerze mniej kodu? To tylko sugestia ;)

Reply:
jestem zaaa 😉